test/passing/tests/label_option_default_args.ml.ref

```diff
@@ -68,10 +68,10 @@ let (* 0 *) f
 let f ?l:(C x) = e
 
 let (* 0 *) f
-    ?l:(
-      (* 1 *)
-      (* 2 *)
-      (* 3 *) C (* 4 *) x (* 5 *) ) =
+    ?l:((* 1 *)
+        (* 2 *)
+        (* 3 *)
+      C (* 4 *) x (* 5 *) ) =
```

Comment:
Missing an indentation here ?

Reply:
This is not directly due to this PR (it just made it visible), I commited a fix in this PR but it can also be extracted in another PR

Reply:
The fix is small and related so we can keep them together.